packages/web3torrent/src/components/torrent-info/channels-list/ChannelsList.tsx

```diff
@@ -9,7 +9,7 @@ import {WebTorrentContext} from '../../../clients/web3torrent-client';
 export type UploadInfoProps = {
   wires: PaidStreamingWire[];
   channels: Dictionary<ChannelState>;
-  pseType: 'seeder' | 'leecher';
+  participantType: 'payer' | 'beneficiary';
```

Comment:
Wondering whether `data-sender` and `data-receiver` would be even clearer?

Reply:
This component is about Channels primarily, though. And these are the terms we are using in the `PaymentChannelClient`.